frontend/src/metabase/browse/components/utils.tsx

```diff
@@ -39,36 +49,36 @@ export const getSecondarySortColumn = (
   return sort_column === "name" ? "collection" : "name";
 };
 
-export const sortModels = (
-  models: ModelResult[],
+export function sortCards<T extends ModelResult | MetricResult>(
+  metrics: T[],
```

Comment:
Seems like we need a term that covers both models or metrics. Is "card" the emerging term? Thinking out loud: maybe (though it is ugly) "modelOrMetric", for the sake of explicitness?

Using 'metrics' here may confuse because of the function's other use.